recipes/breakpad/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+    "cci.20210521":
+        folder: all
```

```suggestion
  "cci.20210521":
    folder: all
```
Some bike shedding: use an indentation of 2 chars in `.yml`'s :P